### ISSUE TITLE
Warning: package version create is not a sf command.

### ISSUE THREAD

**Issue by dschach** [forcedotcom/cli#1963]
### Summary

`Warning: package version create is not a sf command.`

### Steps To Reproduce:

**Repository to reproduce:** [ActionPlansV4](https://github.com/dschach/ActionPlansV4)

The script [build](https://github.com/dschach/ActionPlansV4/blob/main/scripts/packagedev/build.sh) uses a command that is explicitly allowed in the documentation: [sf package version create](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_package_commands_unified.htm#cli_reference_package_version_create_unified) but I get the error that it is not a sf command.

### Expected result

The sf CLI should be consistent with documentation.

### Actual result

Error shown as above

### System Information

- Which shell/terminal are you using? (e.g. bash, zsh, powershell 5, powershell 7, cmd.exe, etc.)

```json

{
  "cliVersion": "@salesforce/cli/1.65.0",
  "architecture": "darwin-arm64",
  "nodeVersion": "node-v18.13.0",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 1.4.5 (core)",
    "@oclif/plugin-commands 2.2.7 (core)",
    "@oclif/plugin-help 5.2.4 (core)",
    "@oclif/plugin-not-found 2.3.18 (core)",
    "@oclif/plugin-plugins 2.3.0 (core)",
    "@oclif/plugin-search 0.0.13 (core)",
    "@oclif/plugin-update 3.1.3 (core)",
    "@oclif/plugin-version 1.2.1 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.26 (core)",
    "@oclif/plugin-which 2.2.14 (core)",
    "community 2.2.1 (core)",
    "custom-metadata 2.1.1 (core)",
    "data 2.2.1 (core)",
    "deploy-retrieve 1.8.0 (core)",
    "env 2.0.2 (core)",
    "functions 1.17.4 (core)",
    "generate 1.1.2 (core)",
    "info 2.4.2 (core)",
    "limits 2.3.3 (core)",
    "login 1.1.18 (core)",
    "org 2.3.2 (core)",
    "schema 2.3.1 (core)",
    "settings 1.2.1 (core)",
    "signups 1.4.0 (core)",
    "sobject 0.1.2 (core)",
    "telemetry 2.1.1 (core)",
    "trust 2.4.1 (core)",
    "user 2.3.2 (core)"
  ],
  "osVersion": "Darwin 22.3.0",
  "shell": "zsh",
  "rootPath": "/Users/<me>/.local/share/sfdx/client/7.188.1-4a4bb9c/sf"
}
```

**Comment by dschach** [comment]
Update: When I run `sfdx force:package:version:create` I get this:

```
Warning: The "force package version create" command has been deprecated. Use "package version create" instead.
Warning: The "--codecoverage" flag has been deprecated. Use "--code-coverage" instead.
Warning: The "--installationkeybypass" flag has been deprecated. Use "--installation-key-bypass" instead.
```

It looks like there are errors, no matter what I do. And I don't see where "--code-coverage" is specified in the [sfdx command section](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_force_package.htm#cli_reference_force_package_version_create)

**Comment by peternhale** [comment]
@dschach we are in the process of restructuring commands. The warnings you are seeing are to help you move to the new structure, but at your own pace. The warnings you see should not affect the execution of the command and you should be able to use the new command and flag names today.

The other ongoing work we are doing is making all the commands available in sfdx available in sf. The packaging commands were included in [v1.66.2](https://github.com/forcedotcom/cli/blob/main/releasenotes/sf/README.md#1662-feb-22-2023-stable) of sf.

**Comment by peternhale** [comment]
I will make a note of the discrepancies you are seeing in the sfdx command guide. We apologize for the confusion caused by this.
